### DISCUSSION TITLE
Flutter speed index for flutter boundary

### DISCUSSION THREAD

**Discussion by jjywow** [su2code/SU2#1965] [Q&A]
Hello, Thank you in advance for your concern.

I often encountered papers presenting flutter boundary plots with Mach number versus flutter speed index as below.

Given an airfoil shape and Mach number, how can I obtain the flutter speed index as output in SU2?

Is there a specific solver for the flutter speed index?

How can I set the flutter speed index as output?

Or how can I get eigenvalues for stability eigenvalue problem for given aerodynamic conditions

Any guidance or tips will be appreciated!

Thank you for your help!

![flutter](https://user-images.githubusercontent.com/102700363/224861928-af7ea0ef-02a4-4ce1-b901-9037730b38f0.png)
Sanchez, R., Kline, H.L., Thomas, D., Variyar, A., Righi, M., Economon, T.D., Alonso, J.J., Palacios, R., Dimitriadis, G. and Terrapon, V., 2016. Assessment of the fluid-structure interaction capabilities for aeronautical applications of the open-source solver SU2.

![flutterboundary](https://user-images.githubusercontent.com/102700363/224856654-a607cf12-4e3a-4f59-93f5-25a251e4fafa.png)
Opgenoord, M.M., Drela, M. and Willcox, K.E., 2018. Physics-based low-order model for transonic flutter prediction. AIAA Journal, 56(4), pp.1519-1531.

**Comment by bigfooted** [comment]
I do not know anything about flutter, but there is a configuration keyword FLUTTER_SPEED_INDEX and an example in the testcases folder Testcases/aeroelastic/aeroelastic_NACA64A010.cfg:

https://github.com/su2code/SU2/tree/master/TestCases/aeroelastic
https://github.com/su2code/TestCases/tree/master/aeroelastic

**Reply by jjywow** [reply]
Thank you for looking into this!
